keyboards/hardlineworks/otd_plus/config.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED  // "HLW"
```

Comment:
You want to do something to match the comment here, and avoid `0xFEED`
```suggestion
#define VENDOR_ID       0x484C  // "HL"
```